package.json

```diff
@@ -449,7 +449,7 @@
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.11.1",
     "webpack-node-externals": "^3.0.0",
-    "xterm": "^4.19.0",
+    "xterm": "^5.0.0",
```

Comment:
This commit removes the last usage for earlier introduce image-snapshot dependency. Could you remove it?

Reply:
Sure also